MonoGame.Framework/Input/KeyboardUtil.OpenTK.cs

```diff
@@ -1,343 +1,352 @@
                     return Keys.F5;
-                    
+
                 case OpenTK.Input.Key.F6:
                     return Keys.F6;
-                    
+
                 case OpenTK.Input.Key.F7:
                     return Keys.F7;
-                    
+
                 case OpenTK.Input.Key.F8:
                     return Keys.F8;
-                    
+
                 case OpenTK.Input.Key.F9:
                     return Keys.F9;
-                    
+
                 case OpenTK.Input.Key.G:
                     return Keys.G;
-                    
+
                 case OpenTK.Input.Key.H:
                     return Keys.H;
-                    
+
                 case OpenTK.Input.Key.Home:
                     return Keys.Home;
-                    
+
                 case OpenTK.Input.Key.I:
                     return Keys.I;
-                    
+
                 case OpenTK.Input.Key.Insert:
                     return Keys.Insert;
-                    
+
                 case OpenTK.Input.Key.J:
                     return Keys.J;
-                    
+
                 case OpenTK.Input.Key.K:
                     return Keys.K;
-                    
+
                 case OpenTK.Input.Key.Keypad0:
                     return Keys.NumPad0;
-                    
+
                 case OpenTK.Input.Key.Keypad1:
                     return Keys.NumPad1;
-                    
+
                 case OpenTK.Input.Key.Keypad2:
                     return Keys.NumPad2;
-                    
+
                 case OpenTK.Input.Key.Keypad3:
                     return Keys.NumPad3;
-                    
+
                 case OpenTK.Input.Key.Keypad4:
                     return Keys.NumPad4;
-                    
+
                 case OpenTK.Input.Key.Keypad5:
                     return Keys.NumPad5;
-                    
+
                 case OpenTK.Input.Key.Keypad6:
                     return Keys.NumPad6;
-                    
+
                 case OpenTK.Input.Key.Keypad7:
                     return Keys.NumPad7;
-                    
+
                 case OpenTK.Input.Key.Keypad8:
                     return Keys.NumPad8;
-                    
+
                 case OpenTK.Input.Key.Keypad9:
                     return Keys.NumPad9;
-                    
+
                 case OpenTK.Input.Key.KeypadAdd:
                     return Keys.Add;
-                    
+
                 case OpenTK.Input.Key.KeypadDecimal:
                     return Keys.Decimal;
-                    
+
                 case OpenTK.Input.Key.KeypadDivide:
                     return Keys.Divide;
-                    
+
                 case OpenTK.Input.Key.KeypadEnter:
                     return Keys.Enter;
-                    
+
                 case OpenTK.Input.Key.KeypadMinus:
-                    return Keys.OemMinus;
-                    
+                    return Keys.Subtract;
```

Comment:
I just verified this here too.  I get OemMinus for the key between the numbers and backspace.  I get Subtract for the key on the numpad.